packages/mwp-app-server/src/index.js

```diff
@@ -37,9 +41,15 @@ export default function start(languageRenderers, { routes = [], plugins = [] })
 	const baseRoutes = getRoutes();
 	const finalRoutes = [...routes, ...baseRoutes];
 
-	const connection = {
+	const serverConfig = {
 		host: '0.0.0.0',
 		port: appConfig.app_server.port,
+		app: appConfig,
+	};
+
+	/*
+	* TODO: figure out what these are for
```

Comment:
from dr.mike. need to figure out the v17 equivalent.
```
for `routes.plugins`, it provides per-route plugin config, which can be accessed with `request.route.settings.plugins`, e.g. here https://github.com/meetup/meetup-web-platform/blob/master/packages/mwp-logger-plugin/src/index.js#L20
packages/mwp-logger-plugin/src/index.js:20
    ```const routePluginSettings = route.settings.plugins['mwp-logger-plugin'];```
meetup/meetup-web-platformAdded by GitHub

but yeah i don’t see specific docs about that either so it might be either a deprecated feature or just an undocumented one. I think I just copied that approach from a third-party plugin we’re using

```